Objects/EventItem/Mag.tscn

```diff
@@ -0,0 +1,22 @@
+[gd_scene load_steps=3 format=2]
+
+[ext_resource path="res://Objects/EventItem/Mag.gd" type="Script" id=1]
```

Comment:
`Mag` is polish name. I think, we should name everything in the same language, so `Wizard` will be better ;) (change script name as well)

Reply:
mag seems not so Polish :P but is and i changed this